pipelines/utils/secret.py

```
@@ -15,6 +18,8 @@ def get_secret(secret_path: str = "/", secret_name: str = None, environment: str
     Returns:
         dict: Dicionário com os dados retornados do Infisical
     """
+    log(os.environ["webhooks"])
```

Ruff (0.12.2)

Use capitalized environment variable `WEBHOOKS` instead of `webhooks`

Replace `webhooks` with `WEBHOOKS`

(SIM112)